instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const CommentText = styled.div`
```

Comment:
Wow, first time seeing it styled this way!

Reply:
It's called Styled Components. It's covered by Lambda now (I think CS9 didn't get it but CS11 on did). I think it's covered by the lecture next Wednesday!

Reply:
Speaking of, we did indeed just cover it! 🔥